atomic_reactor/util.py

```diff
@@ -666,6 +666,10 @@ def get_manifest(
         except (HTTPError, RetryError) as ex:
             if ex.response is None:
                 raise
+            if ex.response.status_code == requests.codes.unauthorized:
+                logger.warning('Requested parent/base image "%s" not found', image.to_str())
+                raise RuntimeError('Unable to fetch image: "{}"'
```

Comment:
I don't think that it should be raising here, it should act same as not_found IMHO

Reply:
I am not sure that's ideal because the `saved_not_found` exception is only examined during digests, i.e. https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/util.py#L720 during other calls it is simply ignored(https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/util.py#L757, https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/util.py#L784).

it's quite confusing TBH, that's why I chose to do this in this function only rather than relying on other functions I don't fully understand

Reply:
yeah I know that saved_not_found is later ignored, my idea was to propagate that up and not ignore that, but I guess it would need more refactoring